src/Http/Routing/WebRouting.php

```diff
@@ -12,13 +15,28 @@ public function wire(): void
     {
         $this->router->get("/", fn() => view("welcome"))->name("home");
 
-        $this->router->controller(MeetupController::class)->middleware("auth")->group(function (): void {
-            $this->router->get("/meetups", "index")->name("meetups");
-            $this->router->get("/meetups/create", "create")->name("meetups.create");
-            $this->router->post("/meetups", "store")->name("meetups");
-            $this->router->get("/meetups/{meetup}/edit", "edit")->name("meetups.edit");
-            $this->router->put("/meetups/{meetup}", "update")->name("meetups.update");
-            $this->router->delete("/meetups/{meetup}", "destroy")->name("meetups.destroy");
+        $this->router->prefix("/auth")->group(function (): void {
+            $this->router->get("/register", [RegisterController::class, "create"])->name("register.form");
+            $this->router->post("/register", [RegisterController::class, "store"])->name("register");
+            $this->router->get("/login", [LoginController::class, "store"])->name("login.form");
+            $this->router->post("/login", [LoginController::class, "login"])->name("login");
+            $this->router->get("/logout", [LoginController::class, "logout"])->name("logout")->middleware("auth");
+
+            $this->router->controller(SocialiteController::class)->group(function (): void {
+                $this->router->get("/google/redirect", [SocialiteController::class, "redirectToGoogle"])->name("login.google");
```

Comment:
You've already added `controller()` method for this group. Repeating controller's name in every route is unneccesarry.